database.js

```diff
@@ -0,0 +1,17 @@
+require('dotenv').config();
+var knex = require('knex')({
+  client: 'mssql',
+  connection: {
+    host: process.env.DB_HOST,
+    user: process.env.DB_USER,
+    password: process.env.DB_PASS,
+    database: process.env.DB_DATABASE
+  }
+});
+
+knex.schema.createTable(process.env.TABLE_NAME, function(table) {
```

Comment:
You don't need to do this here. It's mainly to hide sensitive details like username, password or a key to an api and whatnot.

Reply:
ohhh is it? I thought it is mainly for placing all variables into one file instead of hard-coding them into multiple places.
Cool, will remove them it now. :)